index.js

```diff
@@ -144,18 +144,6 @@ module.exports = {
     "import/no-named-default": "error",
     "import/no-webpack-loader-syntax": "error",
     "import/no-useless-path-segments": ["error", { noUselessIndex: true }],
-    "import/order": [
```

Comment:
Just to be sure, the rule is not activated in any preset ? So dropping it from here is enough ?